src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -159,15 +160,20 @@ public static void NearMinMaxDateTimeOffsetConvertTest()
         VerifyConvert(new DateTimeOffset(DateTime.MinValue.AddHours(5), new TimeSpan(-3, 0, 0)), s_strPacific, new DateTimeOffset(DateTime.MinValue, new TimeSpan(-8, 0, 0)));
 
         VerifyConvert(DateTime.MaxValue, s_strPacific, s_strSydney, DateTime.MaxValue);
-        VerifyConvert(DateTime.MaxValue.AddHours(-19), s_strPacific, s_strSydney, DateTime.MaxValue);
+        if (Interop.IsWindows) // [ActiveIssue(846, PlatformID.AnyUnix)]
```

Comment:
This is blocked by the issue I raised in email regarding how to handle AdjustmentRules on Linux.

We are not handling daylight savings time after 2038 correctly for Australia/Sydney because
1. We are not reading the V2 version information.
2. We aren't taking into account the POSIX style rule on how to handle daylight savings after the last transition in the file: i.e. - AEST-10AEDT,M10.1.0,M4.1.0/3, so everything after 2038 stays on one offset for now.

I plan on fixing this issue in my next change after this round of PRs.